### ISSUE TITLE
out of bounds for type iw_channelinfo_out

### ISSUE THREAD

**Issue by asarubbo** [jsummers/imageworsener#21]
On 1.3.1 with -fsanitize=undefined:

```
# imagew $FILE /tmp/out -outfmt bmp
$FILE -> /tmp//out
Processing: 7x32514
Warning: This image may have transparency, which is incompatible with the output format. A background color will be applied.
src/imagew-main.c:960:12: runtime error: index -1 out of bounds for type 'struct iw_channelinfo_out [4]'
SUMMARY: AddressSanitizer: undefined-behavior src/imagew-main.c:960:12 in 
imagew error: Compression type incompatible with image type

```

Reproducer:
https://github.com/asarubbo/poc/blob/master/00280-imageworsener-oob-iw_channelinfo_out

**Comment by jsummers** [comment]
Should be fixed by commit a4f247707f.

**Comment by rshariffdeen** [comment]
CVE-2017-9203 has been assigned
